go/host/p2p/ttl_map.go

```diff
@@ -0,0 +1,63 @@
+package p2p
+
+import (
+	"sync"
+	"time"
+)
+
+type item struct {
+	value      int64
+	lastUpdate time.Time
+}
+
+// ttlMap is a map in which the values are removed after a certain amount of time
```

Comment:
Maybe worth explaining the behaviour in a bit more detail here. From reading the code I think (but am not sure): the map stores an incrementing count per key but those counts reset themselves to zero if not incremented within the configured TTL.

Reply:
Yes that's correct. Added a more thorough description.